src/com/sap/piper/Utils.groovy

```diff
@@ -7,19 +7,31 @@ import groovy.text.GStringTemplateEngine
 import java.nio.charset.StandardCharsets
 import java.security.MessageDigest
 
+def stash(Map params) {
+    if(params.includes == null) params.includes = '**/*.*'
+    if(params.excludes == null) params.excludes = ''
+    if(params.useDefaultExcludes == null) params.useDefaultExcludes = true
+    if(params.allowEmpty == null) params.allowEmpty = false
```

Comment:
We will have less trouble if `allowEmpty` is true by default. Does an empty stash harm (... in case no value for `allowEmpty` was provided? Do we expect violating the fail-fast approach when allow empty stashes and find problems with missing files later?

Reply:
Jenkins default is `allowEmpty = false` (we should double-check!).
This means the behaviour of `Utils.stash` until now was allowEmpty=false since we did not pass the parameter to the Jenkins `stash` step.